salt/metalk8s/saltmaster/files/salt-master-pod.yaml.j2

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: saltmaster
```

Comment:
I'd use `salt-master`, but maybe that's bikeshedding :) Also then applies to labels etc.

Reply:
done